README.md

```diff
@@ -28,6 +28,19 @@ pip install -r requirements.txt
 # Try it out
 python manim.py example_scenes.py OpeningManimExample
 ```
+### Mac OSX
+1. Install FFmpeg,LaTeX,cairo in terminal using homebrew.
```

Comment:
Please add a space after each comma, it will look better:
1. Install FFmpeg, LaTeX, cairo in terminal using homebrew.